cli/pom.xml

```diff
@@ -729,19 +729,11 @@
             </exclusions>
         </dependency>
 
+        <!-- Dependency needed by the CLI fat jar (client classifier) to support JBoss Logging JSON Formatter 
```

Comment:
Minor nit, it's not JBoss Logging that requires it, but the JBoss Log Manager.

Reply:
Thank-you. Fixed